mir-ci/mir_ci/tests/robot/suites/fractional_scale_v1/fractional_scale_v1.robot

```diff
@@ -0,0 +1,14 @@
+*** Settings ***
+Resource        ${KVM_RESOURCE}
+
+*** Variables ***
+${T}    ${CURDIR}
+
+*** Test Cases ***
+Click Button With Fractional Scaling Enabled
+    Log to console          ${KVM_RESOURCE}
+    Match                   ${T}/gtk4-demo-title-app-title.png
+    Move Pointer To         ${T}/gtk4-demo-button-simple-constraints.png
```

Comment:
This keyword has embedded arguments:

https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#embedding-arguments-into-keyword-name

```suggestion
    Move Pointer To ${T}/gtk4-demo-button-simple-constraints.png
```

When you look at [the keyword documentation](https://github.com/canonical/mir-ci/blob/main/mir-ci/mir_ci/tests/robot/resources/kvm/kvm.resource), anywhere you see "$<foo>" in the keyword name itself, you don't separate it - that only works for those with `[Arguments]`.